v1/api/db.py

```diff
@@ -0,0 +1,394 @@
+            conn.close()
+
+        return status, error
+
+    @staticmethod
+    def delete_diary_entry(eid):
+        """ Deletes a diary entry """
+        status = False
+        error = ''
+
+        query = "DELETE FROM diary WHERE eid={}".format(eid)
+        select = "SELECT * FROM diary WHERE eid={}".format(eid)
+
+        dbm = DatabaseManager()
+        conn = dbm.connect_db()
+
+        try:
+            cur = conn.cursor()
+
+            cur.execute(select)
+            record = cur.fetchone()
+
+            if not record:
+                error = "no entry with selected id found"
+                return status, error
+
+            cur.execute(query)
+            cur.close()
+            conn.commit()
+            status = True
+        except psycopg2.DatabaseError as ex:
+            print(ex.pgerror)
+            error = str(ex.pgerror)
+        finally:
+            conn.close()
+
+        return status, error
+
+    @staticmethod
+    def create_tables():
+        """ This method creates the tables in which the API stores data """
+
+        commands = ("""
+            CREATE TABLE IF NOT EXISTS users
+(
+    uid serial,
+    firstname varchar(30) NOT NULL,
+    lastname varchar(30) NOT NULL,
+    username varchar(30) NOT NULL UNIQUE,
+    email varchar(100) NOT NULL UNIQUE,
+    password varchar(160) NOT NULL,
+    PRIMARY KEY (uid)
+    );""", """CREATE TABLE IF NOT EXISTS diary
+(
+    eid serial,
+    uid integer NOT NULL,
+    title varchar(200) NOT NULL,
+    content text NOT NULL,
+    created timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (eid),
+    FOREIGN KEY(uid) REFERENCES users(uid) ON UPDATE NO ACTION ON DELETE CASCADE
+); """, """CREATE TABLE IF NOT EXISTS notifications
+(
+    nid serial,
+    uid integer NOT NULL,
+    created date NOT NULL DEFAULT CURRENT_DATE,
+    frequency integer,
+    PRIMARY KEY (nid, uid),
+    FOREIGN KEY(uid) REFERENCES users(uid) ON UPDATE NO ACTION ON DELETE CASCADE
+); """)
+
+        dbm = DatabaseManager()
+        conn = dbm.connect_db()
+
+        try:
+            cur = conn.cursor()
+
+            for sql_command in commands:
+                cur.execute(sql_command)
+
+            cur.close()
+            conn.commit()
+
+        except Exception as ex:
+            print(ex)
+        finally:
+            conn.close()
+
+    @staticmethod
+    def drop_tables():
+        commands = ("DROP TABLE IF EXISTS notifications",
+                    "DROP TABLE IF EXISTS diary", "DROP TABLE IF EXISTS users")
+
+        dbm = DatabaseManager()
+        conn = dbm.connect_db()
+
+        try:
+            cur = conn.cursor()
+
+            for command in commands:
+                cur.execute(command)
+
+            cur.close()
+            conn.commit()
+
+        except Exception as ex:
+            print(ex)
+        finally:
+            conn.close()
+
+
+if __name__ == '__main__':
+   # DatabaseManager.drop_tables()
+    DatabaseManager.create_tables()
```

Comment:
change your static methods to instance methods and reference your connect_db method with self to avoid class instance creation with in your methods.